web/src/js/__tests__/components/Header/__snapshots__/FileMenuSpec.js.snap

```diff
@@ -72,17 +72,9 @@ exports[`FileMenu Component should render correctly 1`] = `
         />
          Options
       </a>
-       
-    </li>
-    <li>
```

Comment:
FWIW I don't think it's the rendering algorithm - looks more like a bug to me. There clearly should be a `<li>` here generated by this: https://github.com/mitmproxy/mitmproxy/blob/6d9b5b0c062126d2ac47fa50795c7d560321e0f7/web/src/js/components/common/Dropdown.jsx#L49

Does the file menu still look the same? If so, I think we can tolerate this for now.